### PR TITLE
MediaPlayer continued

### PR DESCRIPTION
**Pull request by mitchdowney**

### CODE REVIEW

File: Podverse/AppDelegate.swift [hunk 1]
```diff
@@ -101,13 +101,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func skipBackwardEvent() {
-        PVMediaPlayer.shared.previousTime(seconds: 15)
-        PVMediaPlayer.shared.setPlayingInfo()
+        if let currentItem = pvMediaPlayer.avPlayer.currentItem {
+            pvMediaPlayer.goToTime(seconds: CMTimeGetSeconds(currentItem.currentTime()) - 15)
```
Comment (kreonjr) [inline Podverse/AppDelegate.swift:105]: Why not call setPlayingInfo() directly in the goToTime() method of the PVMediaPlayer instead? It is always being called after a goToTime from what it seems like.

File: Podverse/AppDelegate.swift [hunk 2]
```diff
@@ -101,13 +101,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+        pvMediaPlayer.setPlayingInfo()
     }
     
     func skipForwardEvent() {
-        PVMediaPlayer.shared.skipTime(seconds: 15)
-        PVMediaPlayer.shared.setPlayingInfo()
+        if let currentItem = pvMediaPlayer.avPlayer.currentItem {
+            pvMediaPlayer.goToTime(seconds: CMTimeGetSeconds(currentItem.currentTime()) + 15)
+        }
+        pvMediaPlayer.setPlayingInfo()
```
Comment (kreonjr) [inline Podverse/AppDelegate.swift]: Same as above

File: Podverse/PVMediaPlayer.swift [hunk 3]
```diff
@@ -19,21 +19,21 @@ enum PlayingSpeed {
         get {
             switch self {
             case .Quarter:
```
Comment (kreonjr) [inline Podverse/PVMediaPlayer.swift]: New Swift 3 standard. enum cases should always start with lowercase. (ie .quarter)

**Review by kreonjr — CHANGES_REQUESTED** [review]

File: Podverse/AppDelegate.swift (hunk 1 above)
Comment (mitchdowney) [reply Podverse/AppDelegate.swift:105]: Actually, I'm not sure we even need setPlayingInfo on time changes. I'll remove it for now and put it back if it turns out we need it.

**Review by kreonjr — APPROVED** [review]

**Pull request merged**